src/status_im2/contexts/onboarding/new_to_status/view.cljs

```diff
@@ -110,19 +110,20 @@
                              :icon-background :blur
                              :icon            :i/arrow-left
                              :on-press        navigate-back}
-     :right-section-buttons [(when config/quo-preview-enabled?
-                               {:type            :grey
-                                :icon            :i/reveal-whitelist
-                                :icon-background :blur
-                                :on-press        #(rf/dispatch [:navigate-to
-                                                                :quo2-preview])})
-                             {:type            :grey
-                              :icon            :i/info
-                              :icon-background :blur
-                              :on-press        #(rf/dispatch
-                                                 [:show-bottom-sheet
-                                                  {:content getting-started-doc
-                                                   :shell?  true}])}]}]])
+     :right-section-buttons (cond-> [{:type            :grey
```

Comment:
The more future proof solution is to change `right-section-buttons` to deal with nils (just filter them out), since it's quite common to use `when` to build up vectors. For example, re-frame even mentions this as one of the features of its `:fx` effect. 

> if any element of :fx is nil, it will be ignored. This allows effects to be conditionally included via a when (an example is given above)
>
> -- https://day8.github.io/re-frame/releases/2020/

Although `cond->` is just fine, the better fix imo is to make `page-nav` more resilient.

Reply:
> if any element of :fx is nil, it will be ignored. This allows effects to be conditionally included via a when (an > example is given above)

This is really helpful. Thanks for sharing!

I agree @ilmotta. We should this filter in all the component which receives a vector to prevent these types of crashes.

I faced this same issue when I was refactoring the action-drawer component, I added a filter to prevent it.
https://github.com/status-im/status-mobile/blob/e5b6378e1575f12c91de1db829395625613bac4a/src/quo2/components/drawers/action_drawers/view.cljs#L98

> the better fix imo is to make page-nav more resilient

I will add the filter in a separate PR 👍

Reply:
@ulisesmac is looking into that component I believe :)

Reply:
Yes, I will improve our page-nav component 👍